javascript/javascript2/week1/Week1HW.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+//Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const shortestWord = getShortestWord(danishWords); // returns 'ø'
+function getShortestWord(words) {
+  let newWord = danishWords[0];
+  for (let i = 0; i < danishWords.length; i++) {
+    if (danishWords[i].length <= newWord.length) {
+      newWord = danishWords[i];
+      danishWords[i] = danishWords[0];
+      danishWords[0] = newWord;
+    }
+  }
+  return danishWords[0];
+}
+document.getElementById("ShortestWord").innerHTML =
+  "shortest word in array : " + shortestWord;
+console.log(
+  'Shortest word in array ["bil", "plante", "kaffe", "bog", "ø", "planetarium"] is : ' +
+    shortestWord
+);
+
+//Find and count the Danish letters
+const danishString2 = "Blå grød med røde bær";
+let danishLetters = danishLettersCounter(danishString2);
+function danishLettersCounter(danString) {
+  const stringArray = danString.split(" ");
+  // console.log(stringArray);
+  let total = 0;
+  let åCount = 0;
+  let æCount = 0;
+  let øCount = 0;
+  for (let i = 0; i < stringArray.length; i++) {
+    if (stringArray[i].includes("å")) {
```

Comment:
Why not `===` instead of `includes`? Whats the different?

Reply:
in here for split , I have used split(" ") which it compares words and in this way === doesn't work because it compares words to letter ! but I will change it to split("") with no space then === also works !